Core/Services/Auth/ExternalLoginService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Core.Interfaces.Auth;
+using Core.ServiceResponses;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Storage.Identity;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Core.Services.Auth
+{
+    class ExternalLoginService : AuthServicesProvider, IExternalLoginService
```

Comment:
Is the default internal access modifier correct for this class?

Reply:
Yes, because I use this class only in this folder